### PR TITLE
test(execute): add tests to confirm that yield supports null

### LINKED ISSUE #644: Update yield to understand Null

### PR DESCRIPTION
**Pull request by affo**
Fixes #644 

### Done checklist
- [x] docs/SPEC.md updated
- [x] Test cases written

### CODE REVIEW

File: execute/executor_test.go [hunk 1]
```diff
@@ -643,6 +643,57 @@ func TestExecutor_Execute(t *testing.T) {
...
+								{nil, execute.Time(5), execute.Time(0), nil, nil, true},
+								{execute.Time(0), nil, execute.Time(1), nil, "t0", nil},
+								{execute.Time(0), nil, execute.Time(2), 3.0, "t0", nil},
+								{execute.Time(0), execute.Time(5), execute.Time(3), nil, "t0", false},
+								{execute.Time(0), nil, execute.Time(4), nil, "t0", true},
+							},
+						}},
+					)),
+					plan.CreatePhysicalNode("yield", &universe.YieldProcedureSpec{Name: "_result"}),
+				},
```
[the remark this answers is not in the document]
Comment (nathanielc) [reply execute/executor_test.go]: While trivial its a valid test :D
